exercise-1/script.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('button');
+const nav = document.querySelector('nav');
+const main = document.querySelector('.hide-main');
```

Comment:
Same as in CSS, try to be a bit more specific. As soon as you start adding other buttons etc. to the page, this script will run into issues.